test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -33,7 +33,7 @@
     private Logic logic;
 
     @Before
-    public void setup() throws Exception {
```

Comment:
setup is a single word, no need to make it upper case

Reply:
Hi Adoby7!

Thanks for your time and effort reviewing my PR.

However, please simply search on the Web so that you know more about the JUnit Framework.

“*setup*” can become a single word in English. But, in JUnit, it is a commonly-used method, which should be written as `setUp`.

In future, you may want to improve your coding style by looking into the Codacy report.

For example, you can see [here](https://www.codacy.com/app/se-edu/addressbook-level3/file/10046029907/issues/source?bid=3942469&fileBranchId=3942469#l36) to identify why we need to change it to `setUp`

![1](https://user-images.githubusercontent.com/18503402/30519317-f494e406-9bc5-11e7-89b1-426422ccbe30.png)

Hope this can help you.

Regards,
Yunpeng